tutorials/Qa_skullstrip.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+ "cells": [
```

Comment:
Your tutorial does nothing to explain what inputs to give your qa, what is happening in your qa functions, or what the outputs mean. Someone new to this program will have no idea what is going on. You MUST edit this

Reply:
going to fix it in the latest version

Reply:
https://github.com/NeuroDataDesign/ndmg/blob/qa_skullstrip_pr/tutorials/Qa_skullstrip.ipynb